src/Employees/Api/EmployeesController.cs

```diff
@@ -51,8 +52,9 @@ public async Task<ActionResult<EmployeeExtendedJson>> GetExtendedByAlias(string
         }
 
         var emergencyContact = await _employeeService.GetEmergencyContactByEmployee(employee);
+        var allergiesAndDietaryPreferences = await _employeeService.GetAllergiesAndDietaryPreferencesByEmployee(employee);
 
-        return ModelConverters.ToEmployeeExtendedJson(employee, emergencyContact);
+        return ModelConverters.ToEmployeeExtendedJson(employee, emergencyContact, allergiesAndDietaryPreferences);

```

Comment:
Vi må passe litt på at denne ikke bare vokser og vokser jo mer man legger til, kanskje det burde være små spesialiserte endepunkt for hver ting heller. ⚠️

Reply:
Jeg kan forstå det om det skulle bli en veldig kompleks profil med flere sider, men er samtidig at den oppfatning at det er gunstig å ha få requests istedenfor mange små? Her tenker jeg også det er logisk å sende alt i en siden det er til en spesifikk side og all dataen skal vises samtidig. Hadde vært enig i at det ville være logisk å dele det opp i flere om det var f.eks. tabs eller ligennde slik at ikke all dataen ville vises samtidig. Hvor tenker du grensen går mellom å ha en og å dele dem opp?

Reply:
Tenker vi er innafor nå, ikke over grensa, så derfor jeg bare skrev det som en generell ting å ta med videre